.github/workflows/wheels-build.yml

```diff
@@ -173,16 +173,13 @@ jobs:
   wheels-linux-aarch64:
     name: "Wheels / Linux AArch64"
     if: (inputs.wheels-linux-aarch64 == 'default' && inputs.default-action || inputs.wheels-linux-aarch64) == 'build'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ inputs.python-version }}
       - uses: dtolnay/rust-toolchain@stable
```

Comment:
I can't comment on L186 below but I think we can remove that line and use the default test command. Tier 2 we define as running the tests during build/release but not on PR CI. We limited it to just `test.python.compiler.test_transpiler` in the interest of runtime because the emulation was so slow. But if we have a native host we should have the time budget to run the full test suite which is more in spirit of what we have defined as tier 2 support.

Reply:
Done in 2b394e4099, let's see what happens.